.ci/containers/go-ruby-python/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM gcr.io/magic-modules/go-ruby:1.11.5-2.6.0-v2
+FROM gcr.io/magic-modules/go-ruby:1.11.5-2.6.0-v4
```

Comment:
Is it possible to just use the `latest` tag on these? Or if not `latest` some other tag that indicates "this is the current production build image"?

Reply:
I think we've avoided doing this because it makes testing harder (I want to push up a image for testing, but it picks up the latest tag, so everybody has to use my testing image).

There's so little drift on the Magician now that I think we could safely use a `latest` tag because it's a pain dealing with these images.

Reply:
When we talked about this before, we decided, we should use `latest`. Images pushed for testing should not use the `latest` tag. `latest` is a commonly-used tag, new revs pushed do not automatically use `latest`.

https://github.com/GoogleCloudPlatform/magic-modules/issues/2049 tracks consolidating images + using `latest` (I haven't taken the time to do it yet, if you want to pick up either / both parts feel free)

Reply:
I must have been subliminally thinking of that issue when I commented :-) Since we have a tracking issue, LGTM.